users/tests/test_user_login_logout.py

```diff
@@ -40,11 +43,21 @@ def test_sign_in_unexistant_email(self):
 
     def test_sign_out_success(self):
         self.client.login(email=self.user.email, password=self.password)
-        response = self.client.post('/dj-rest-auth/logout/')
+        response = self.client.post(reverse('rest_logout'))
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertTrue('detail' in response.data)
 
+    def test_login_without_confirmation(self):
+        ea = EmailAddress.objects.get()
+        ea.verified = False
+        ea.save()
+        data = {'email': self.user.email, 'password': self.password}
```

Comment:
Just a tip, maybe this is not actually the case:
If you want to get a dict from a factory object, you can do

`data_dict = factory.build(dict, FACTORY_CLASS=MyFactoryClass)`